### PR TITLE
fix: add flow test fix

### PR DESCRIPTION
**Pull request by subirjolly**
Remove `Link to Panel` from side for querybuilder or figure out why it's not showing up in deployment pipeline.

### CODE REVIEW

**Review by asalem1 — COMMENTED** [review]
How does this resolve anything?

**Review by asalem1 — APPROVED** [review]
🤞🏽

**Comment by subirjolly** [comment]
if this doesn't fix then i will add a network wait for each panel in the next PR

**Review by asalem1 — APPROVED** [review]
I've never seen conditional statements for intercepting so we'll see if that works

**Comment by subirjolly** [comment]
> 

I tested these locally and work fine. but yes, not sure if this is actually going to fix the issue.

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 52 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=4015)

**Comment by subirjolly** [comment]
> I've never seen conditional statements for intercepting so we'll see if that works

if you want, then I can divide this loop into individual assertion groups..

File: cypress/e2e/shared/flows.test.ts [hunk 1]
```diff
@@ -528,17 +529,32 @@ describe('Flows', () => {
...
-      cy.getByTestID(panelTestId).click()
+      if (item.panel === 'queryBuilder') {
+        cy.getByTestID(panelTestId).click()
+        cy.wait('@fetchAllBuckets')
+      } else if (item.panel === 'notification') {
+        cy.getByTestID(panelTestId).click()
+        cy.wait('@fetchSecrets')
+      } else {
+        cy.getByTestID(panelTestId).click()
+      }
```
Comment (asalem1) [inline cypress/e2e/shared/flows.test.ts:556]: we can probably simplify this with two ifs for the waits with the click before it right?

cy.click()
if (query) wait
if (notification) wait

**Pull request merged**